pkg/handlers/v1/eni.go

```diff
@@ -51,7 +51,7 @@ func (t eniTransformer) Create(event awsConfigEvent) (Output, error) {
 
 // Returns true if we should filter this event due to not being requester managed
 func filter(config eniConfiguration) bool {
-	return config.RequesterManaged == false || config.RequesterID != elbRequester
+	return !config.RequesterManaged || config.RequesterID != elbRequester
```

Comment:
a bit pedantic of go lint to require this but as long were consistent with it, I am fine with using the `!` to check for false here.